ethcore/src/trace/db.rs

```diff
@@ -305,7 +305,7 @@ impl<T> TraceDatabase for TraceDB<T> where T: DatabaseExtras {
 	}
 
 	fn trace(&self, block_number: BlockNumber, tx_position: usize, trace_position: Vec<usize>) -> Option<LocalizedTrace> {
-		let trace_position_deq = trace_position.into_iter().collect();
+		let trace_position_deq = trace_position.into_iter().collect::<VecDeque<usize>>();
```

Comment:
:+1: Good point. Had to touch this line since it wasn't compiling on latest nightly.